plugins/network/multibandwidth

```diff
@@ -101,11 +104,20 @@ for host in $hosts; do
                 | cut -d "b" -f1)
 
         if (echo "$SPEED" | grep -q "M"); then
-                echo "$SPEED" | awk '{a+=$1} END{print a*1000000}'
+                VALUE=`echo "$SPEED" | sed 's/.$//'`
+                RATE=`echo "$VALUE * 1048576" | bc -l`
```

Comment:
Please use `$(...)` instead of backticks. This allows nested execution and is easier to read.
In this specific case you can even use arithmetic expansion (and avoid the dependency on `bc`):
```
RATE=$((VALUE * 1048676))
```